LandSandBoat/server
# [lua] Fix enhancing magic that depends on skill for potency, update enhancing magic params table to have ticks

**_I affirm:_**

## What does this pull request do?

Fix enhancing magic that depends on skill for potency, update enhancing magic params table to have ticks

## Steps to test these changes

use Phalanx, !getmod Phalanx should be non-zero
use Foil, !getmod SPECIAL_ATTACK_EVASION should tick down
